<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="HomeAlarmButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>258</y>
     <width>400</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>闹钟</string>
   </property>
  </widget>
  <widget class="QLabel" name="HomeTimeDisplayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HomeWthPlaceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>258</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>地点</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HomeDateTimeBackLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>18</y>
     <width>160</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeImagesButton">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>18</y>
     <width>350</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HomeWthInfoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>天气</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HomeDateDisplayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>118</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeSettingButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>448</y>
     <width>400</width>
     <height>128</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="HomeTemInfoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>468</y>
     <width>120</width>
     <height>88</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HomeTemBackLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>448</y>
     <width>160</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeMediaButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>18</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>200</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>多媒体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeWeatherButton">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>258</y>
     <width>350</width>
     <height>318</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>800</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>天气预报</string>
   </property>
  </widget>
  <zorder>HomeWeatherButton</zorder>
  <zorder>HomeTemBackLabel</zorder>
  <zorder>HomeDateTimeBackLabel</zorder>
  <zorder>HomeAlarmButton</zorder>
  <zorder>HomeTimeDisplayLabel</zorder>
  <zorder>HomeImagesButton</zorder>
  <zorder>HomeDateDisplayLabel</zorder>
  <zorder>HomeSettingButton</zorder>
  <zorder>HomeTemInfoLabel</zorder>
  <zorder>HomeMediaButton</zorder>
  <zorder>HomeWthInfoLabel</zorder>
  <zorder>HomeWthPlaceLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
